src/inspector/view/qml/MuseScore/Inspector/general/GeneralInspectorView.qml

```diff
@@ -143,7 +143,7 @@ InspectorSectionView {
 
                 PlaybackPopup {
                     id: playbackPopup
-                    navigation.parentControl: playbackButton.navigation
+                    navigationParentControl: playbackButton.navigation
```

Comment:
Is it necessary to set parental control here? Because you set the previous control as a parent for the popup by default in PopupView::open

Reply:
You right, not necessary, but I prefer to set explicit.